Userland/Applications/TextEditor/MainWidget.cpp

```diff
@@ -653,6 +653,10 @@ bool MainWidget::open_file(const String& path)
         return false;
     }
 
+    if (file->is_directory()) {
+        GUI::MessageBox::show(window(), String::formatted("Opening \"{}\" failed: Can't open directores", path), "Error", GUI::MessageBox::Type::Error);
```

Comment:
Typo, and you have to actually return false here. 
```suggestion
        GUI::MessageBox::show(window(), String::formatted("Opening \"{}\" failed: Can't open directories", path), "Error", GUI::MessageBox::Type::Error);
        return false;
```

Reply:
Thanks! I'll fix that right now